indra/llcommon/llerror.cpp

```diff
@@ -1602,10 +1602,15 @@ namespace LLError
         return out << boost::stacktrace::stacktrace();
     }
 
-    // LLOutOfMemoryWarning
-    std::string LLUserWarningMsg::sLocalizedOutOfMemoryTitle;
-    std::string LLUserWarningMsg::sLocalizedOutOfMemoryWarning;
-    LLUserWarningMsg::Handler LLUserWarningMsg::sHandler;
+    // LLOutOfMemoryWarning, LLOutOfDiskWarning
+    // need to be preallocated before viewer runs out of memory/disk
+    static std::string sLocalizedOutOfMemoryTitle;
```

Comment:
sLocalizedOutOfMemoryTitle and showOutOfMemory() were preallocated (arguably should have been inline as well) because it's out of memory handling, to not allocate once we run out. There is no point prerecording these strings for everything else.